pkg/endpoint/api.go

```diff
@@ -76,6 +76,7 @@ func NewEndpointFromChangeModel(ctx context.Context, owner regeneration.Owner, p
 	ep.dockerEndpointID = base.DockerEndpointID
 	ep.K8sPodName = base.K8sPodName
 	ep.K8sNamespace = base.K8sNamespace
+	ep.K8sUID = base.K8sUID
```

Comment:
Do we need to consider this field during endpoint restore serialization/deserialization in endpoint (i.e. `(*Endpoint).toSerializedEndpoint` and `(*Endpoint).ep.fromSerializedEndpoint`) as well?

Reply:
Yes!

Reply:
Thanks, good catch!